lib/player.rb

```diff
@@ -0,0 +1,8 @@
+class Player
+  attr_reader :name, :tag
+
+  def initialize(name = 'Player X', tag = 'X')
```

Comment:
Interface design understandability: It feels a little unsymmetrical for the default values to be useable in one half of the cases.  How about a factory method that creates the appropriate player eg `Player.create(:o)`?